spiffworkflow-backend/tests/spiffworkflow_backend/unit/test_process_instance_migrator.py

```diff
@@ -111,7 +111,7 @@ def test_can_run_version_4_migration(
         processor = ProcessInstanceProcessor(
             process_instance, include_task_data_for_completed_tasks=True, include_completed_subprocesses=True
         )
-        bpmn_process_dict_version_4 = processor.serialize()
+        bpmn_process_dict_version_4 = processor.serialize(serialize_script_engine_state=False)
```

Comment:
Tests are passing for me fine with default settings. Is it actually necessary? I don't think the current changes should require this.

Reply:
I edited the description, which upon a re-read wasn't very clear, but these changes are to get the migrator tests passing with the non task data based environment. This was part of the work on the `non_task_data_env` branch to get the unit tests passing.